user-console/src/main/resources/org/pentaho/mantle/public/browser/js/browser.js

```diff
@@ -1180,9 +1229,9 @@ define([
         var path = $target.attr("path");
         var myself = this;
 
-        var url = CONTEXT_PATH + "api/repo/files/" +
-            FileBrowser.encodePathComponents(path == null ? ":" : Encoder.encodeRepositoryPath(path))
-            + "/tree?depth=1&showHidden=" + myself.model.get("showHiddenFiles") + "&filter=*%7CFOLDERS";
+        var url = CONTEXT_PATH + "plugin/scheduler-plugin/api/generic-files/files/" +
+            FileBrowser.encodePathComponents(path == null ? ":" : path.replaceAll(':','~').replaceAll('/',':'))
```

Comment:
Could have a the following method:
```js
_encodeGenericPath: function(path) {
  return path.replaceAll(":", "~").replaceAll("/", ":");
},
```

Still need to pass the result through `FileBrowser.encodePathComponents`.

Regarding path being `null`... if we're expanding a folder, then there must be a path. And the root, root path is loaded upfront, not in this code branch.